test/behavior/export_builtin.zig

```diff
@@ -57,7 +54,10 @@ test "exporting using field access" {
 test "exporting comptime-known value" {
     if (builtin.zig_backend == .stage2_wasm) return error.SkipZigTest;
     if (builtin.zig_backend == .stage2_arm) return error.SkipZigTest;
-    if (builtin.zig_backend == .stage2_x86_64) return error.SkipZigTest;
+    if (builtin.zig_backend == .stage2_x86_64 and
+        (builtin.target.ofmt != .elf and
+        builtin.target.ofmt != .macho and
+        builtin.target.ofmt != .coff)) return error.SkipZigTest;
```

Comment:
I think you can remove these lines, right?

Reply:
I can swap it out for `builtin.zig_backend == .stage2_x86_64 and builtin.target.ofmt == .p9` since that one is still unimplemented.

Reply:
I see - never mind, I think it's good how you wrote it :+1: